src/containers/Interfaces/__tests__/CategoricalBin.test.js

```diff
@@ -15,7 +15,7 @@ describe('CategoricalBin', () => {
   it('renders CategoricalBin interface', () => {
     const component = shallow(<CategoricalBin {...requiredProps} />);
     expect(component.find('.categorical-bin-interface')).toHaveLength(1);
-    expect(component.find('.categorical-bin-interface').children()).toHaveLength(4);
+    expect(component.find('.categorical-bin-interface').children()).toHaveLength(3);
```

Comment:
I have _no idea_ why this was ever 4. Can't see anything in the commit history about there being another child element in this component. Why did it ever pass!?

Side note: I don't feel bad about removing tests for things like "renders correctly", since we can move these to the integration tests now, which will be more meaningful.

Reply:
I changed this one to a snapshot test since I think that better captures the essence of what this test is for. 

I think the advantage of these tests over the integration tests is just that they run that much faster, which is useful for local development - but you are right we could remove it.